src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: $trans-speed;
```

Comment:
it's better to specify the transition property
say
`transition: color $trans-speed;` etc.